Formula/gwyddion.rb

```diff
@@ -25,7 +26,6 @@ class Gwyddion < Formula
   depends_on "fftw"
   depends_on "gtk+"
   depends_on "gtkglext"
-  depends_on "gtksourceview"
```

Comment:
As I mentioned in extended commit message, this commit drops some features to avoid deprecating `gwyddion` for now. This would temporarily extend lifespan of `gwyddion` as there are a couple GTK 2 formulae that have upstream trackers/commits for GTK 3 support, so we will probably leave `gtk+` undeprecated for a few more months.

Alternative would be to keep dependency on `gtksourceview` and deprecate `gwyddion` now. This would retain feature set, but has higher risk of broken bottles as we will no longer dependent test the formula.

GTK 2 (`gtk+`) dependency is mandatory, but all other GTK 2 features (`gtkglext`, `gtksourceview`, `gtk-mac-integration`) are optional: http://gwyddion.net/documentation/user-guide-en/installation-dependencies.html